.github/workflows/test-r-scripts.yaml

```diff
@@ -1,4 +1,12 @@
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    branches:
+      - main
+      - master
```

Comment:
By being more specific, we can now push non-pr branches and not trigger CI -- to save computation and time.